packages/components/src/VirtualizedList/CellTitle/CellTitleActions.scss

```diff
@@ -0,0 +1,28 @@
+@import '../colors';
+$tc-list-title-actions-color: $brand-primary-dark !default;
+
+.main-title-actions-group {
+	flex-grow: 0;
+	position: relative;
+	right: 0;
+	top: 0;
+
+	:global(.tc-actions) {
+		&:not(.persistent-actions) {
+			opacity: 0;
```

Comment:
Why playin with opacity if it's not visible?

Reply:
I just moved this code, but I presume that the `opacity` is here to be able to animate the action's apparition, and the `visibility` is here for accessibility reasons (screen reader, for example).